src/mixins/audit.mixin.ts

```diff
@@ -127,8 +128,7 @@ export function AuditRepositoryMixin<
           data =>
             new AuditLog({
               actedAt: new Date(),
-              // eslint-disable-next-line @typescript-eslint/no-explicit-any
-              actor: (user?.id as any).toString() ?? '0', //NOSONAR
+              actor: this.getActor(user, this.actorIdKey, options?.actorId),
```

Comment:
arre why do you need to pass this.actorIdKey, the getActor would have access to it na as that is a member variable?

Reply:
rectified